lib/dot-json.js

```diff
@@ -56,7 +56,7 @@ module.exports = function(file) {
 				_this._indent = detectIndent(_this.content()).indent;
 			}
 			catch (e) {
-				_this._indent = 2
+				_this._indent = '  '
```

Comment:
The tests were alternating failure, probably depending on whether the file existed or not.

This function now always returns a string, never a number.

CI passes now: https://github.com/fregante/dot-json/commits/ci